docs/sdk-beta/introduction.md

```diff
@@ -0,0 +1,26 @@
+# DroneDeploy App SDK (beta)
+
+### Note that this is a beta and things are subject to change.
+
+## Overview
+
+Welcome to the DroneDeploy App SDK Beta. We are very excited to share with you what we have been working on to help you make the most out of the DroneDeploy Platform. By participating in this beta, you will have access to the latest features of the DroneDeploy Platform and also have an impact and influence over the product roadmap.
+
+The new App SDK introduces three main features in addition to the existing UI platform:
+
+![](overview.png)
+
+
+* [**UI**](../getting_started.md): Build a custom user interface
+
+* [**Functions**](functions.md): Serverless architecture running on the DroneDeploy Platform. Write Node.js code to enable backend functionality.
```

Comment:
These descriptions for Functions, Datastore and Triggers are different than those on the `beta-signup` page.  Is that intentional?

Reply:
yes intentional